dts/bindings/pinctrl/nxp,s32ze-pinctrl.yaml

```diff
@@ -83,6 +81,8 @@ child-binding:
           - slew-rate
           - input-enable
           - output-enable
+          - current-reference-control
+          - termination-resistor
```

Comment:
these are not valid properties of `pincfg-node.yaml`

Reply:
i removed it, thank